Automattic/jetpack
# Add support for v.wordpress.com URLs in VideoPress embed block

Fixes Automattic/greenhouse#1068

#### Changes proposed in this Pull Request:

This PR adds an oembed provider for v.wordpress.com URLs and updates the VideoPress core/embed block to match these URLs.

#### Does this pull request change what data or activity we track or use?
No.

#### Notes

This PR is based on fix/merge-videopress-block-in-video-block, I'll update the PR to merge in master once the other branch is merged.

#### Testing instructions:

* On a Jetpack site with a sandboxed wpcom public API
* Apply the wpcom patch D72345-code
* Edit or create a new post and add a /video block (and/or a /videopress embed block if visible)
* Click on "Insert from URL"
* Paste a v.wordpress.com embed URL (like http://v.wordpress.com/KllQxFVq )
* ✅ The embed should work and display the VideoPress player
